### DISCUSSION TITLE
Can C developers contribute w/ new features?

### DISCUSSION THREAD

**Discussion by federico-r-figueredo** [tursodatabase/libsql#168] [General]
Greetings,
                Wanted to query if new features will be implemented in C? Or there's a preference on them to be implemented in Rust? Because I'd like to contribute but I'm not interested on learning (nor using) Rust. I want to continue using C.

Best Regards.

**Comment by psarna** [comment]
@federico-r-figueredo Rust is a preference, but we've already accepted contributions in C and we'll continue to do so! (Maybe until 100% of the code base is Rust, but we're long way from there)
